### ISSUE TITLE
cptac 1.5.10 pandas

### ISSUE THREAD

**Issue by cchen22** [PayneLab/cptac#61]
I got a weird error when importing cptac 1.5.10 with `import cptac`. cptac 1.5.9 has the same issue but cptac 1.5.8 works fine.
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[1], line 1
----> 1 import cptac
      2 import cptac.utils as ut

File ~\AppData\Local\anaconda3\envs\cptac_env\Lib\site-packages\cptac\__init__.py:83
     79     # options_df.loc[options_df.iloc[:2].str.contains('miRNA')] = 'miRNA' # condense all forms of micro RNA
     80     # options_df = options_df.unique().reset_index(drop=True)
     81     return options_df
---> 83 OPTIONS = _load_options()
     85 def list_datasets(*, condense_on = None, column_order = None, print_tree=False):
     86     """
     87     List all available datasets.
     88     
     89     :param condense_on (list): A list of column names. Values in selected columns will be aggregated into a list.
     90     :param print_tree (bool): If True, returns the database split in a pretty tree.
     91     """

File ~\AppData\Local\anaconda3\envs\cptac_env\Lib\site-packages\cptac\__init__.py:77, in _load_options()
     75 """Load the tsv file with all the possible cancer, source, datatype combinations"""
     76 options_df = pd.DataFrame(INDEX['description'].str.split('-').tolist())
---> 77 options_df.columns = ['Source', 'Cancer', 'Datatype']
     78 options_df = options_df[['Cancer', 'Source', 'Datatype']]
     79 # options_df.loc[options_df.iloc[:2].str.contains('miRNA')] = 'miRNA' # condense all forms of micro RNA
     80 # options_df = options_df.unique().reset_index(drop=True)

File ~\AppData\Local\anaconda3\envs\cptac_env\Lib\site-packages\pandas\core\generic.py:6002, in NDFrame.__setattr__(self, name, value)
   6000 try:
   6001     object.__getattribute__(self, name)
-> 6002     return object.__setattr__(self, name, value)
   6003 except AttributeError:
   6004     pass

File ~\AppData\Local\anaconda3\envs\cptac_env\Lib\site-packages\pandas\_libs\properties.pyx:69, in pandas._libs.properties.AxisProperty.__set__()

File ~\AppData\Local\anaconda3\envs\cptac_env\Lib\site-packages\pandas\core\generic.py:730, in NDFrame._set_axis(self, axis, labels)
    725 """
    726 This is called from the cython code when we set the `index` attribute
    727 directly, e.g. `series.index = [1, 2, 3]`.
    728 """
    729 labels = ensure_index(labels)
--> 730 self._mgr.set_axis(axis, labels)
    731 self._clear_item_cache()

File ~\AppData\Local\anaconda3\envs\cptac_env\Lib\site-packages\pandas\core\internals\managers.py:225, in BaseBlockManager.set_axis(self, axis, new_labels)
    223 def set_axis(self, axis: AxisInt, new_labels: Index) -> None:
    224     # Caller is responsible for ensuring we have an Index object.
--> 225     self._validate_set_axis(axis, new_labels)
    226     self.axes[axis] = new_labels

File ~\AppData\Local\anaconda3\envs\cptac_env\Lib\site-packages\pandas\core\internals\base.py:70, in DataManager._validate_set_axis(self, axis, new_labels)
     67     pass
     69 elif new_len != old_len:
---> 70     raise ValueError(
     71         f"Length mismatch: Expected axis has {old_len} elements, new "
     72         f"values have {new_len} elements"
     73     )

ValueError: Length mismatch: Expected axis has 1 elements, new values have 3 elements
```

**Comment by sgosline** [comment]
This is the same error I'm getting in #60

**Comment by sabmel** [comment]
I apologize for the inconvenience! We just released a version 1.5.11 that should fix this issue, so please try pip install once again or --upgrade. Please let us know if the issue persists!
